doc-Managing_Infrastructure_and_Inventory/topics/Instance_Evacuation.adoc

```diff
@@ -1,4 +1,4 @@
-If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/instances-and-images-guide/chapter-3-virtual-machine-instances#section-migration-evacuation[Evacuate Instance] (from the https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/instances-and-images-guide/instances-and-images-guide[Red Hat OpenStack Platform Instances and Images Guide]) for details on the underlying OpenStack process.
+If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] in the Red Hat OpenStack Platform _Instances and Images Guide_ for details on the underlying OpenStack process.
```

Comment:
Fixed in: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0